### ISSUE TITLE
something wrong with select_team or login

### ISSUE THREAD

**Issue by vancex** [kaich/spaceship#1]
All is right but no "app" api when I used 0.21.0 version,  but after I updated to 0.25.1 version,  I login an apple id  which used to generate free ceriticate by xocde,  it can not get teams any more, reponse is just "[]" in the log file.
 but version 0.21.0 is ok, even more I can add device udid by api successsfully!

**Comment by kaich** [comment]
Post all your error log here, please.

**Comment by vancex** [comment]
is it different between ur spaceship and fastlane?

**Comment by kaich** [comment]
yes,  there is many different with fastlane/spaceship. Many code is edited by myself. and you can't install my spaceship and fastlane/spaceship at same time. I have consider to edit my code and pull request to fastlane  in my rest time.  before this , you can edit my code as you like! 
I'm curious to know what do you want to do with this code.

**Comment by vancex** [comment]
I mailed u, let's talk my purpose there~
